src/main/java/org/folio/linked/data/service/resource/ResourceServiceImpl.java

```diff
@@ -103,6 +105,10 @@ private Resource saveNewResource(ResourceRequestDto resourceDto, Resource old) {
     var mapped = resourceDtoMapper.toEntity(resourceDto);
     metadataService.ensure(mapped, old.getFolioMetadata());
     resourceEdgeService.copyOutgoingEdges(old, mapped);
+    mapped.setCreatedDate(old.getCreatedDate());
+    mapped.setVersion(old.getVersion() + 1);
+    mapped.setCreatedBy(old.getCreatedBy());
+    mapped.setUpdatedBy(folioExecutionContext.getUserId());
```

Comment:
Shouldn't we set updatedDate here with the current date & time?
Consider this scenario:
Day 1: A new resource created using POST API or import
Day 2: The resource is updated through PUT API. The updatedDate should contain the current date.

Reply:
updated_date will contain the current date, ```@UpdateTimestamp``` will do this for us.